default/hypr/bindings/tiling.conf

```diff
@@ -7,6 +7,10 @@ bindd = SUPER, P, Pseudo window, pseudo, # dwindle
 bindd = SUPER, V, Toggle floating, togglefloating,
 bindd = SHIFT, F11, Force full screen, fullscreen, 0
 
+# Group windows
+bind = SUPER SHIFT, T, Toggle tabs, togglegroup
+bind = SUPER, Tab, Cycle to next window in tabs, changegroupactive
```

Comment:
are you sure it's not `bindd`? 

using `bind` will get `invalid dispatcher error`